polling_stations/apps/data_collection/base_importers.py

```diff
@@ -657,9 +657,12 @@ def get_station_point(self, record):
                 srid=27700)
         else:
             # otherwise, geocode using postcode
+            postcode = getattr(record, self.station_postcode_field).strip()
+            if postcode == '':
```

Comment:
Should this be `if not postcode`? It's possible that the data might parse in to `0` or `None` in some cases rather than an empty string. `not postcode` will call python's `bool` on `postcode` and catch all of these cases I think?